src/components/views/rooms/MessageComposerInput.js

```diff
@@ -868,7 +861,7 @@ export default class MessageComposerInput extends React.Component {
             return true;
         }
 
-        const newState: ?Value = null;
+        //const newState: ?Value = null;
```

Comment:
Maybe remove this?

Reply:
The block of code that uses it is commented below, so I figured it should be left as long as the commented out block is there :/